packages/react-core/src/components/Dropdown/DropdownToggle.tsx

```diff
@@ -18,6 +18,8 @@ export interface DropdownToggleProps extends React.HTMLProps<HTMLButtonElement>
   onToggle?: (isOpen: boolean) => void;
   /** Element which wraps toggle */
   parentRef?: HTMLElement;
+  /** The menu element */
+  getMenuRef?: any;
```

Comment:
More of a question here. Is this expected to be a function?  To me the name implies function. If it is a function maybe update description.  If not maybe update name to `menuRef`?  
Is the any type to allow for Element or function?